Makefile

```diff
@@ -228,3 +228,6 @@ test-docs: build-docs
 serve-docs: build-docs
 	@echo Stating docs website on http://localhost:${DOCS_PORT}/_build/html/index.html
 	@docker run -i --rm -p ${DOCS_PORT}:8000 -e USER_ID=$$UID ignite-docs
+
+e2e: $(BINARIES)
```

Comment:
This might need to be `build-all` instead, because we should be testing with a fresh OCI image for `ignite-spawn`

Reply:
oh right. this should be fix to cover the case when `ignite-spawn` is changed.
thank you @stealthybox !